spec/unit/pops/loaders/loaders_spec.rb

```diff
@@ -107,7 +107,7 @@
     end
 
     it 'loader allows loading a function more than once' do
-      File.stubs(:read).with(user_metadata_path).returns ''
+      Puppet::FileSystem.stubs(:read).with(user_metadata_path, {:encoding => 'utf-8'}).returns ''
```

Comment:
@hlindberg this PR has triggered a number of test failures in `loaders_spec.rb` that I haven't had time to fully chase down yet.. wondering if you quickly have an idea on how to address the test failures.  If you don't know OTOH, no big deal - I'll eventually track them down, but thought it might be worth checking before I get to that point.